### ISSUE TITLE
this.set() is undefined — contradicts ReadMe example

### ISSUE THREAD

**Issue by christophermlne** [pangratz/ember-cli-htmlbars-inline-precompile#14]
In the example code (see below) taken from the readme, there is a call to this.set().  

My understanding is that `this` will be set to the context of the test when the test is running. I don't see .set() defined on the test object when I toss in a debugger statement just before that line.  

How does one pass data to an inline-precompiled hbs template? I've tried declaring the var in multiple places, hoping it would be grabbed by the closure somehow. Nothing seems to work.

I am using Ember 1.12.1  
ember-qunit 0.3.3
   

    import hbs from 'htmlbars-inline-precompile';
    import { moduleForComponent, test } from 'ember-qunit';

    moduleForComponent('my-component', {
      integration: true
    });

    test('block params work', function(assert) {
      this.render(hbs`
        {{#my-component date=theDate as |daysAgo| }}
          This happened {{daysAgo}} days ago.
        {{/my-component}}
      `);

      this.set('theDate', new Date(2015, 2, 11));
      assert.equal(this.$().text().trim(), "This happened 123 days ago.");
    });

**Comment by pangratz** [comment]
Hey! `this.set` should definitely work when using `integration: true` and `ember-qunit >= 0.3.2`. Do you have a a sample repo where the issue can be reproduced?

**Comment by pangratz** [comment]
In 776a8e5ec1a797758d88e340561fb429acfe3644 I added a test for block parameters using this plugin. The test is similar to the one in the README. Do you spot any difference to the setup you have in your app/addon?

**Comment by christophermlne** [comment]
thanks for taking the time to comment.

weird. i was about to follow up with more detail, but this.set() is suddenly working. perhaps i just needed to restart ember cli? the data i'm passing in is still not being rendered by the template, but that is a separate issue.

thanks again.

**Comment by matthewconstantine** [comment]
Since this seemed unsolved and I ended up here in my quest to solve the missing `this.set` here's what the cause of the problem was for me.

I had something like this:
```moduleForComponent('x-foo', {
  integration: true,
  needs: ['helper:pluralize-string']    // <--- wrong
});```

Turns out `ember-qunit` silently forces your integration test into a unit test when `needs` is present. Remove it and it becomes an integration test again and `this.set` is available.
